### PR TITLE
Add additional requirements for production deployments.

### PR DESCRIPTION
**Pull request by jasonthomas**

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -17,3 +17,9 @@ konfig==0.8
 SQLAlchemy==0.8.2
 MySQL-python==1.2.4
 umemcache==1.6.3
+importlib==1.0.2
```
Comment (almet) [inline requirements/prod.txt:20]: Importlib causes problems with python 2.7 (see https://travis-ci.org/mozilla/addon-registration/jobs/13705228#L126).

Comment (andymckay) [inline requirements/prod.txt:20]: https://github.com/mozilla/zamboni/blob/master/requirements/prod.txt#L55

importlib-no-failure has no issue with python 2.7

Comment (almet) [inline requirements/prod.txt:20]: thanks for pointing that out.

**Comment by almet** [comment]
Updated the commit https://github.com/mozilla/addon-registration/commit/75c54b6ccc8765345565af745e9e28fe4a3d59c8 and merged into master.

**Pull request closed without merging**